build.gradle

```diff
@@ -30,23 +30,24 @@ repositories {
 }
 
 configurations {
-    // We're using Jetty instead
-    compile.exclude module: "spring-boot-starter-tomcat"
     // We're using HikariCP instead
     compile.exclude module: "tomcat-jdbc"
 }
 
 dependencies {
     compile("org.springframework.boot:spring-boot-starter-data-jpa")
-    compile("org.springframework.boot:spring-boot-starter-jetty")
```

Comment:
Not really, but Tomcat has `javax.validation` bundled which was somehow missing after upgrading to Spring Boot 1.4.0.RELEASE with Jetty.